src/main/java/com/textmessenger/model/entity/dto/UserToFrontShort.java

```diff
@@ -0,0 +1,32 @@
+package com.textmessenger.model.entity.dto;
+
+import com.textmessenger.model.entity.User;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class UserToFrontShort {
+  private long id;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "id" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_bCAJjEPbGfqJZM&open=AWWU5_bCAJjEPbGfqJZM)

[ISSUE_KEY]: AWWU5_bCAJjEPbGfqJZM